README.md

```diff
@@ -105,6 +105,7 @@ opam install dream-html
 Alternatively, to install the latest commit that may not have been released yet:
```

Comment:
```suggestion
Alternatively, to install the latest commit that may not have been released yet, run _one_ of the following depending on which package you need:
```

Reply:
Actually that is not what I meant in this PR. To install dream-html, you need to pin both. Otherwise dream-html will not have its dependency met.

Reply:
Whoops, my bad. You're right. Pushing a change in the wording.

Reply:
The new push pins `https://github.com/yawaramin/pure-html` which is non existent at the moment. Will `pure-html` be separated in its own repository soon?

Reply:
Separating the packages would also get rid of the source of the issue: dream-html and pure-html being tied to the same version, while they really don't need to be.

Reply:
Thanks, fixed both issues now!